src/components/Pool/PairsList/Pair/Pari.styles.ts

```diff
@@ -1,5 +1,5 @@
 import styled from 'styled-components'
-import { Flex } from 'rebass'
+import { Flex } from 'rebass/styled-components'
```

Comment:
why from /styled-components?

Reply:
Rebass for styled-components has less default styles.